src/spurt/workflows/emcf/_settings.py

```diff
@@ -30,7 +34,8 @@ class SolverSettings:
         Scale factor used in computing edge costs for spatial unwrapping.
     """
 
-    worker_count: int = 0
+    s_worker_count: int = 0
+    t_worker_count: int = 1
```

Comment:
The order of these parameters is swapped in the docstring. The default values should be swapped as well (`t_worker_count` should default to 0 and `s_worker_count` should default to 1).

```suggestion
    t_worker_count: int = 0
    s_worker_count: int = 1
```